unit-tests-java8/src/test/java/org/eclipse/collections/test/RichIterableTestCase.java

```diff
@@ -381,6 +400,137 @@ default void RichIterable_containsAllIterable()
         assertTrue(iterable0.containsAllIterable(Lists.immutable.empty()));
     }
 
+    @Test
+    default void RichIterable_flatCollectBoolean()
+    {
+        RichIterable<BooleanBooleanPair> iterable = this.newWith(
+                PrimitiveTuples.pair(true, false),
+                PrimitiveTuples.pair(false, true));
+
+        BooleanArrayList target = new BooleanArrayList();
+        iterable.flatCollectBoolean((Function<BooleanBooleanPair, BooleanIterable>) each -> BooleanArrayList.newListWith(
+                each.getOne(),
+                each.getTwo()), target);
+
+        assertSize(4, target);
```

Comment:
It seems like "target" is the equivalent of "expected" in this class (see example method: RichIterable_select_reject). But the usage in here makes me believe that "target" is "actual". 

In short,  you would want this to be "result" and not "target" to be consistent with the rest of the class. 
"result" -> "actual"
"target" -> "expected" 
e.g. assertEquals( expected, actual)